solvers/xgboost.py

```diff
@@ -1,20 +1,20 @@
-from benchopt import safe_import_context
-from benchmark_utils.gridsearch_solver import GSSolver
-
-with safe_import_context() as import_ctx:
-    from xgboost import XGBClassifier
-
-
-class Solver(GSSolver):
-
-    name = 'XGBoost'
-
-    requirements = ['py-xgboost']
-
-    parameter_grid = {
-        'n_estimators': [100, 1000, 2000],
-        'max_depth': range(1, 12)
-    }
-
-    def get_model(self):
-        return XGBClassifier()
+from benchopt import safe_import_context
+from benchmark_utils.gridsearch_solver import GSSolver
+
+with safe_import_context() as import_ctx:
+    from xgboost import XGBClassifier
+
+
+class Solver(GSSolver):
+
+    name = 'XGBoost'
+
+    requirements = ['py-xgboost']
+
+    parameter_grid = {
+        'model__n_estimators': [100, 1000, 2000],
+        'model__max_depth': range(1, 12)

```

Comment:
I would not modify this like this but rather modify in `grid_search_solver` the parameter grid to add `model__` to the keys.

Reply:
This will change with PR #7 (once updated)